src/webgpu/api/operation/command_buffer/copyTextureToTexture.spec.ts

```diff
@@ -75,9 +223,92 @@ g.test('t2t_non_compressed_color_formats')
             srcTextureSize: { width: 32, height: 32, depth: 1 },
             dstTextureSize: { width: 63, height: 61, depth: 1 },
           },
+        ])
+      )
+      .combine(poptions('format', kRegularTextureFormats))
+      .combine(
+        poptions('copyBoxOffsets', [
```

Comment:
Most of this list is a subset of the other, right? I think you should factor it out into a `const` and then use `...copyBoxOffsetsForWholeDepth,` in both.